src/org/zalando/stups/mint/storage/api.clj

```diff
@@ -1,16 +1,16 @@
-; Copyright 2015 Zalando SE
-;
-; Licensed under the Apache License, Version 2.0 (the "License")
-; you may not use this file except in compliance with the License.
-; You may obtain a copy of the License at
-;
-;     http://www.apache.org/licenses/LICENSE-2.0
-;
-; Unless required by applicable law or agreed to in writing, software
-; distributed under the License is distributed on an "AS IS" BASIS,
-; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-; See the License for the specific language governing permissions and
-; limitations under the License.
+;; Copyright 2015 Zalando SE
```

Comment:
any reason for this change?

Reply:
Convention is to use single semicolons for margin comments, my formatter kept indenting these.